src/views/termsofuse/termsofuse.jsx

```diff
@@ -0,0 +1,514 @@
+var React = require('react');
+var render = require('../../lib/render.jsx');
+var Page = require('../../components/page/www/page.jsx');
+var Box = require('../../components/box/box.jsx');
+
+var Termsofuse = React.createClass({
```

Comment:
Would it be better to call this 'Terms' rather than 'Termsofuse'? I'm thinking it might be a little more concise and easier to read.